src/Plugin/StatisticsPluginInterface.php

```diff
@@ -3,12 +3,16 @@
 namespace Drupal\sapi\Plugin;
 
 use Drupal\Component\Plugin\PluginInspectionInterface;
+use Drupal\sapi;
 
 /**
  * Defines an interface for Statistics plugin plugins.
  */
 interface StatisticsPluginInterface extends PluginInspectionInterface {
 
-  // Add get/set methods for your plugin type here.
+  /**
+   * process() method which analize the data received from dispatcher service
+   */
+  public function process(StatisticsItemIterface $item);
```

Comment:
`StatisticsItemIterface` is not imported using `use` statement, therefore the script will fail. I'd suggest adding `use Drupal\sapi\StatisticsItemInterface;`